### ISSUE TITLE
Quick fix for ! nullable operator

### ISSUE THREAD

**Issue by clairernovotny** [dotnet/roslyn#33360]
.NET Core 3 Preview 2

I think it'd help a lot of if Roslyn quick-fix offered to suggest the `!` post-fix operator in addition to the pragmas. It would help discoverability of this new language feature and might even be the default?

Today, it offers the pragmas:
![image](https://user-images.githubusercontent.com/1427284/52733022-a7968880-2f8f-11e9-9fc3-5cfe9615b9a2.png)

This is cleaner:
![image](https://user-images.githubusercontent.com/1427284/52733140-f7754f80-2f8f-11e9-9682-55730f4723c3.png)

**Comment by jasonmalinowski** [comment]
@mavasani Adding the "proper" fix is easy enough -- is there a way to mark the fix as "better" than the suppression (since it's literally a suppression itself!) and not suggest the other stuff?

**Comment by mavasani** [comment]
@jasonmalinowski Yes, that should be possible by either tweaking [PragmWarningCodeAction](http://source.roslyn.io/#Microsoft.CodeAnalysis.Features/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.PragmaWarningCodeAction.cs,b01432ee1bb8d35a) to special case these diagnostics OR changing the logic [here](http://source.roslyn.io/#Microsoft.CodeAnalysis.Features/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.cs,148) to register a different code action instead of PragmaWarningCodeAction for these diagnostics.

**Comment by jasonmalinowski** [comment]
@mavasani But in either case we have to tinker in some core code to add an exception versus some nicer way of saying it? It just feels hacky but I get it.

**Comment by mavasani** [comment]
> some nicer way of saying it?

I don't know of any other way based on the current suppression fixer code.

**Comment by CyrusNajmabadi** [comment]
I don't think we'd do this.  Adding `!` is trivial.
